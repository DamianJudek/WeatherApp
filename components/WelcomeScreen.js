import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  baseText: {
    fontFamily: 'Roboto',
    color: '#fff',
    fontSize: 18,
  },
  heading: {
    fontSize: 35,
  },
});

const Welcome = () => (
  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
    <Text style={[styles.baseText, styles.heading]}>Wyszukaj miasto</Text>
    <Text style={[{ textAlign: 'center' }, styles.baseText]}>lub</Text>
    <Text style={[styles.baseText, styles.heading]}>u≈ºyj lokalizacji</Text>
  </View>
);

export default Welcome;
