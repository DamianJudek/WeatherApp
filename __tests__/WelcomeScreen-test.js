import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import WelcomeScreen from '../components/WelcomeScreen';

test('renders correctly', () => {
  const snap = renderer.create(<WelcomeScreen />).toJSON();
  expect(snap).toMatchSnapshot();
});
