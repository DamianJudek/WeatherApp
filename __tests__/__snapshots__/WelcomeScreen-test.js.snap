// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
          "fontFamily": "Roboto",
          "fontSize": 18,
        },
        Object {
          "fontSize": 35,
        },
      ]
    }
  >
    Wyszukaj miasto
  </Text>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontFamily": "Roboto",
        "fontSize": 18,
      }
    }
  >
    lub
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
          "fontFamily": "Roboto",
          "fontSize": 18,
        },
        Object {
          "fontSize": 35,
        },
      ]
    }
  >
    u≈ºyj lokalizacji
  </Text>
</View>
`;
